src/main/java/com/hms_networks/americas/sc/extensions/historicaldata/HistoricalDataConstants.java

```diff
@@ -61,4 +61,10 @@
 
   /** Default maximum amount of time Historical FIFO can get behind: 48 hours in ms. */
   static final long MAX_HIST_FIFO_GET_BEHIND_MS = 48L * MILLISECONDS_PER_HOUR;
+
+  /** Default thread sleep time waiting for EBD in ms. */
+  static final int DEFAULT_EBD_THREAD_SLEEP_MS = 5;
+
+  /** Maximum amount of time that EBD call should wait before timing out. */
+  static final long MAX_EBD_WAIT = 15000;
```

Comment:
Units should be added to tag name -- i.e. MAX_EBD_WAIT_MILLIS

Reply:
changed